cocoa/events_darwin.go

```diff
@@ -46,6 +46,11 @@ func (w *Window) EventChan() (events <-chan interface{}) {
 	downKeys := make(map[string]bool)
 	ec := make(chan interface{})
 	go func(ec chan<- interface{}) {
+
+		var noX int = 1<<31 - 1
```

Comment:
Would it be okay to set noX to `-1`?  `-1` will always be a invalid value whereas `var noX int = 1<<31 - 1` could very well be valid (although presumably very unlikely).

If yes, would you also like me to update it on the linux side?